ajax/libs/clipboard-polyfill/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "clipboard-polyfill",
+  "filename": "clipboard.min.js",
+  "description": "A polyfill for the asynchronous clipboard API",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lgarron/clipboard-polyfill"
+  },
+  "keywords": [
+    "clipboard",
+    "HTML5",
+    "copy",
+    "copying",
+    "cut",
+    "paste",
+    "execCommand",
+    "setData",
+    "getData",
+    "polyfill"
+  ],
+  "author": "Lucas Garron <code@garron.net> (https://garron.net/)",
+  "license": "MIT",
+  "npmName": "clipboard-js",
```

Comment:
Let's use git auto-update instead. There are more complete versions on git repo.
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/lgarron/clipboard-polyfill.git",
    "fileMap": [
      {
        "basePath": "build",
        "files": [
          "**/!(*.ts)"
        ]
      },
      {
        "basePath": "",
        "files": [
          "clipboard*.js"
        ]
      }
    ]
  }
```

Use "build" as basePath to grab latest versions and use ""(root) as basePath to grab old versions of the lib.